Rules/04 Score/02 Planning & engagement.md

```diff
@@ -8,10 +8,12 @@ But you, the players, don't have to do the nitty-gritty planning. The characters
 
 There are six different plans, each with a missing **detail** you need to provide (see the list below). To "plan an operation," simply choose the plan and supply the detail. Then the GM will **cut to the action** as the first moments of the operation unfold.
 
+## The Plan
+
 * Assault - Do violence to a target. *Detail: The point of attack.*
 * Deception - Lure, trick, or manipulate. *Detail: The method of deception.*
 * Stealth - Trespass unseen. *Detail: The point of infiltration.*
-* Occult - Engage a supernatural power. *Detail: The arcane method.*
+* Alien - Engage an inhuman power. *Detail: The inhuman method.*
```

Comment:
Not attached to "Alien" as the word here, but it seemed like a way to wave at all the stuff we have instead of occult

Reply:
At one point we discussed 'Fae' a sort of post-human faction, could use that as a short hand for heavily biologically modified humans. Supernatural also could be fitting, although may also be imperfect.